site/src/pages/CreateUserPage/CreateUserPage.tsx

```diff
@@ -17,6 +18,25 @@ export const CreateUserPage: FC = () => {
 	const queryClient = useQueryClient();
 	const createUserMutation = useMutation(createUser(queryClient));
 	const authMethodsQuery = useQuery(authMethods());
+	const validatePasswordMutation = useMutation(validatePassword());
+
+	const [passwordValidator, setPasswordValidator] = useState({
```

Comment:
Is there a reason all this logic must live in the parent component instead of directly inside `CreateUserForm`? I could be missing something and @BrunoQuaresma might be able to weigh in.

But if this logic can live in the form component itself, I think it should, for simplicity.

Reply:
Scanning down through the PR, I see that we're duplicating this logic across several views. To avoid doing this, I think we should create a hook called `useValidatePassword` or something similar, and then call that hook across these components.

I suggest pairing with @BrunoQuaresma or @aslilac if you need help here!